lib/drivers/medtronic600/NGPUtil.js

```diff
@@ -123,6 +123,23 @@ class NGPConstants {
     ];
   }
 
+  static get CL_TRANSITION_REASON() {
+    return {
+      INTO_ACTIVE_DUETO_GLUCOSE_SENSOR_CALIBRATION: 0,
+      OUTOF_ACTIVE_DUETO_USER_OVERRIDE: 1,
+      OUTOF_ACTIVE_DUETO_ALARM: 2,
+      OUTOF_ACTIVE_DUETO_TIMEOUT_FRM_SAFE_BASAL: 3,
+      OUTOF_ACTIVE_DUETO_PROLONGED_HIGH_SG: 4,
```

Comment:
Can we make these a bit easier to read by adding `_` between `INTO`, `OUTOF` and `DUETO`? For a moment there I thought "dueto" was Spanish :wink: 

Do we store these values anywhere? I'm guessing they're not supported by the data model yet, which is why Darin is currently working on updating the data model?

Reply:
Yeah, good point. They’re a copy/paste from CLP, but we can do better. TBH, we’re not even using these at the moment. I added them with the intention of using the ClosedLoopTransitionEvent, but then used the isClosedLoop flag instead, since it was more reliable.